cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+    "\n",
+    "    # Zip consensus labels with their corresponding image_paths\n",
+    "    consensus_data = pd.DataFrame(zip(image_paths,consensus_labels), columns=[\"image\", \"label\"])\n",
+    "    \n",
+    "    # Train model\n",
+    "    train_model(model_type, consensus_data, model_xval_results_folder, **train_args)\n",
+    "    pred_probs, labels, images = sum_xval_folds(model_type, model_xval_results_folder, **train_args)\n",
+    "    \n",
+    "    # Get improved consensus labels with label quality multiannotator using model pred probs\n",
+    "    label_quality_multiannotator = get_label_quality_multiannotator(multiannotator_labels, pred_probs, verbose=False)\n",
+    "    consensus_labels = label_quality_multiannotator[\"label_quality\"][\"consensus_label\"].tolist()\n",
+    "    \n",
+    "    unique_indices = len(indices_changed)\n",
+    "    indices_changed.update(list(np.where(consensus_labels != np.array(seen_consensus_labels[-1]))[0]))\n",
+    "    unique_indices = len(indices_changed) - unique_indices\n",
+    "    label_changes_from_prior = np.sum(consensus_labels != np.array(seen_consensus_labels[-1]))\n",
+    "    \n",
+    "    print(\"Num changes in consensus labels from previous itter: \", np.sum(consensus_labels != np.array(seen_consensus_labels[-1])))\n",
+    "    print(\"Num unique indices changed: \", unique_indices)\n",
+    "    # End to delete\n",
+    "    \n",
+    "    results = {\n",
+    "        \"pred_probs\": pred_probs,\n",
+    "        \"consensus_labels_in\": labels, # consensus labels used to train the model\n",
+    "        \"images\": images, \n",
+    "        \"consensus_labels_out\": consensus_labels, # new consensus labels generated from pred_probs\n",
+    "        \"label_changes_from_prior\": label_changes_from_prior, # num changes in consensus labels from previous itterations\n",
+    "        \"unique_indices_added\": unique_indices, # number of unique labels indices changed\n",
+    "    }\n",
+    "    \n",
+    "    model_results[itter] = results\n",
+    "    itter+=1\n",
+    "    \n",
+    "    if unique_indices == 0: # no more label improvement\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9aa26e05",
+   "metadata": {},
+   "source": [
+    "## 4. Measure consensus label accuracy and model performance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99e60410",
+   "metadata": {},
+   "source": [
+    "Since our annotators annotated a dataset `cifar10` to which there exist highly accurate `true_labels`, we can measure and report the accuracy of our methods against the ground truth. \n",
+    "\n",
+    "In a true multiannotator setting this would not be possible since ground truth labels will not exist. Instead we have [benchmarked](link paper or blogpost maybe) this method to ensure this method produces more accurate `consensus_labels` than common inustry methods for getting a consensus."
```

Comment:
"common inustry methods for getting a consensus"
->
"popular existing methods for establishing a consensus label from data labeled by multiple annotators"